app/views/pages/governance/dda.njk

```diff
@@ -42,39 +42,74 @@
+            <li>Providing leadership for the transformation and standardisation of data architecture;</li>
             <li>Delivering semantic interoperability for direct care;</li>
             <li>Acting as an escalation point for issues of Data and Information policies, strategies and standards.</li>
           </ul>
 
           <h2>What is the scope of DDA?</h2>
           <ul>
             <li>Programmes and initiatives overseen by the Digital Delivery Board;</li>
-            <li>Local Health and Care Record Exemplars;</li>
+            <li>Local Health and Care Record Exemplars (LHCREs);</li>
             <li>Other programmes being delivered by NHS Digital;</li>
             <li>As instructed, other programmes being delivered across national Health and
             Social Care bodies where the focus is on national capability or where there is
             a clear interface between national and local capabilities.</li>
           </ul>
+          <div class="nhsuk-grid-row nhsuk-promo-group">
+            <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
 
           <div class="nhsuk-promo nhsuk-promo--small">
             <a class="nhsuk-promo__link-wrapper" href="mailto:ginger.buckland@nhs.net">
               <div class="nhsuk-promo__content">
                 <h3 class="nhsuk-promo__heading">Who do I contact for more information?</h3>
                 <p class="nhsuk-promo__description">
-                Secretariat: Ginger Buckland<br>
+                  Chair:Nicholas Oughtibridge<br>
+                Secretariat: Ginger Buckland
                 </p>
               </div>
             </a>
           </div>
+            </div>
+
+            <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
+              <div class="nhsuk-promo nhsuk-promo--small">
+                <a class="nhsuk-promo__link-wrapper" href="https://hscic365.sharepoint.com/sites/Architecture/ARCHGOVBDS/Lists/Templates/AllItems.aspx">
+                  <div class="nhsuk-promo__content">
+                    <h3 class="nhsuk-promo__heading">The EAB and its sub-group’ decisions are updated by the secretariat here</h3>
+                    <p class="nhsuk-promo__description">
+                      All queries regarding the decisions should be directed to ea-da@nhs.net
+                    </p>
+                  </div>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="nhsuk-grid-row nhsuk-promo-group">
+            <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
+              <div class="nhsuk-promo nhsuk-promo--small">
+                <a class="nhsuk-promo__link-wrapper" href="https://hscic365.sharepoint.com/sites/Architecture/ARCHGOVBDS/Lists/Templates/AllItems.aspx">
+                  <div class="nhsuk-promo__content">
+                    <h3 class="nhsuk-promo__heading">Take me to the documents…</h3>
```

Comment:
"Take me to the documents.." feels a bit strange for a title, this might be need reviewing